src/Components/Missions.js

```diff
@@ -1,8 +1,36 @@
 import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { joinMission } from '../redux/missions/missions';
 
 const Missions = () => {
+  const missions = useSelector((state) => state.missions);
+  const dispatch = useDispatch();
   return (
-    <div>ERROR!! Page Not Found</div>
+    <div className="missions_container">
+      <table>
+        <tbody>
+          <tr>
+            <th>Mission</th>
+            <th>Description</th>
+            <th>Status</th>
+            <th />
+          </tr>
+          {missions &&  missions.map((mission) => (
+          <tr key={mission.mission_id}>
+            <td className="missionName">{mission.mission_name}</td>
+            <td className="missionDescription">{mission.description}</td>
+            <td className={`userBadge ${mission.reserved ? 'member' : ' '}`}><label>
+              {mission.reserved ? 'Active member' : 'NOT A MEMBER'}</label></td>
+            <td className="joinButton">
+              <button onClick={() => dispatch(joinMission(mission.mission_id))} className={`missionBtn ${mission.reserved ? 'joined' : ' '}`}>
```

Comment:
In this line you're creating a new function in every render which can be less performant. You can improve this by moving the function outside the return statement like this:

const handleJoinMission = (missionId) => {
    dispatch(joinMission(missionId));
}
Then, you can use this function inside the onClick event:

<button onClick={() => handleJoinMission(mission.mission_id)} className={`missionBtn ${mission.reserved ? 'joined' : ' '}`}>